nemo/collections/tts/models/fastpitch.py

```diff
@@ -122,6 +126,26 @@ def __init__(self, cfg: DictConfig, trainer: Trainer = None):
         )
         self._input_types = self._output_types = None
 
+
+    def _get_default_text_tokenizer(self):
+        default_text_tokenizer = """
+            text_tokenizer:
+                _target_: nemo.collections.tts.torch.tts_tokenizers.EnglishPhonemesTokenizer
+                punct: true
+                stresses: true
+                chars: true
+                apostrophe: true
+                pad_with_space: true
+                add_blank_at: true
+                g2p:
+                    _target_: nemo.collections.tts.torch.g2ps.EnglishG2p
+                    phoneme_dict: "scripts/tts_dataset_files/cmudict-0.7b_nv22.01"
+                    heteronyms: "scripts/tts_dataset_files/heteronyms-030921"
+                    phoneme_probability: 0.5
+            """
+        return default_text_tokenizer
```

Comment:
Can we make this a dataclass instead?
https://omegaconf.readthedocs.io/en/2.1_branch/structured_config.html#simple-types

Reply:
Yes, make it a data class, then do OmegaConf structured()

Reply:
Done.